### ISSUE TITLE
Simplewallet RPC send produces incorrect unlockTime value and permanently locks coins

### ISSUE THREAD

**Issue by plavirudar** [seredat/karbowanec#25]
I mined a couple of blocks using a fresh compile of karbowanec v1.4.7.775 (78982bd) on Ubuntu 16.04 with gcc6. I then tried to send the mined blocks to an exchange wallet using simplewallet. However, the blocks appear to be stuck on exchange and are not unlocking neither on the exchange nor on the wallet after >100 block confirmations.

The RPC sends were initiated with the following command: 
`curl --data-binary '{"jsonrpc": "2.0", "method": "transfer", "params":{"destinations":[{"amount":1000000000000,"address":"Address"}],"mixin":2,"payment_id":"PaymentID","fee":100000000}}' -H 'content-type: application/json;' http://simplewallet:port/json_rpc`

When I call the command 
`curl --data-binary '{"jsonrpc": "2.0", "method": "get_transfers", "params":{}}' -H 'content-type: application/json;' http://simplewallet:port/json_rpc`, I get the following output: 

The unlock time appears to be a block value 200 years in the future, which appears to mean that the transaction and the change from the UTXOs are now permanently locked for 200 years and therefore lost. 
`{"address":"Address","amount":1000100000000,"blockIndex":203600,"fee":100000000,"output":true,"paymentId":"PaymentID","time":1519613151,"transactionHash":"84308120f276f0315d4ed96016a2036a971a54134fb81087fb51a02d4b981276","unlockTime":27078280},`

This is in contrast to the value of a mined block, which shows the correct unlockTime in blockheight. 
`{"address":"","amount":17560782186951,"blockIndex":203443,"fee":0,"output":false,"paymentId":"","time":1519569849,"transactionHash":"eaa60e4789593c2fb884248bb5a9d804cb60a0be773017febc3aac645db1b3ec","unlockTime":203453},`

TLDR: 

Expected behaviour: RPC call returns a block with a sane unlockTime 10-100 blocks in the future. 

Faulty behaviour: RPC call returns a block 27M blocks in the future.

**Comment by aivve** [comment]
Set  "unlock_time":0 (zero) as described in https://github.com/seredat/karbowanec/wiki/Simplewallet-JSON-RPC-API#transfer-1

You dont want to lock coins sending to the exchange

In CryptoNote protocol if this value is less than CRYPTONOTE_MAX_BLOCK_NUMBER (500000000), then it is interpreted as the block height at which the funds are unlocked https://cryptonote.org/cns/cns004.txt

**Comment by plavirudar** [comment]
Thanks for the quick reply. I have set unlockTime to 0 and now the coins successfully unlock. 

However, this should be set by default or made into a mandatory value, I did not have this issue with other ForkNote coins. Permanently locking all coins that do not have the unlock_time variable set is clearly unexpected behaviour and should be patched.

**Comment by aivve** [comment]
Thanks for suggestion, we need to look how it is implemented in Forknote.

**Comment by aivve** [comment]
Fixed https://github.com/seredat/karbowanec/commit/f72a32a5b9c7367830473930c27aaacaae23230d
